### ISSUE TITLE
CORS Preflight requests are not handler when Access-Control-Request-Headers is not present

### ISSUE THREAD

**Issue by guilherme-santos** [containous/traefik#5896]
### Do you want to request a *feature* or report a *bug*?
Bug

### What did you do?

Running [swagger ui](https://hub.docker.com/r/swaggerapi/swagger-ui/) I'm trying to call an endpoint using DELETE method. My browser is Google Chrome Version 78.0.3904.108.

### What did you expect to see?

Successfully execute the endpoint.

### What did you see instead?

Receive `405 Method Not Allowed` because my backend does not implement the method, I was expecting traefik to respond to the request.

I notice that my request does not send `Access-Control-Request-Headers` and you're verifying if it exists before consider it a PreFlight request https://github.com/containous/traefik/blob/v2.0.5/pkg/middlewares/headers/headers.go#L227

### Output of `traefik version`: (_What version of Traefik are you using?_)

v2.0.5

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

Using k8s config

```yaml
apiVersion: traefik.containo.us/v1alpha1
kind: Middleware
metadata:
  name: cors
spec:
  headers:
    accessControlAllowMethods:
      - "OPTIONS"
      - "GET"
      - "POST"
      - "PUT"
      - "DELETE"
    accessControlAllowHeaders:
      - "Content-Type"
    accessControlAllowOrigin: "origin-list-or-null"
    accessControlMaxAge: 100
    addVaryHeader: "true"
```

**Comment by dtomcej** [comment]
Note:

According to w3, The `Access-Control-Request-Headers` header is ,in fact, optional:

https://www.w3.org/TR/cors/#resource-preflight-requests

**Comment by traefiker** [comment]
Closed by #5903.
